# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/font-matching/reftest.list
# Commit: 3b719cd12933
# Full Hash: 3b719cd1293370d843691b3656b52b04042d5ec4
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-03-02 04:31:47
# Regressor Bug: 1756720
# File Overlap Count: 1
# Description:
#   Bug 1756720 - When using legacy MS Symbol fonts, map character codes from the U+00xx range to the PUA range U+F0xx found in the cmap subtables. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D139724
# ==============================================================================

diff -r 9849c6e79fa3 -r 3b719cd12933 layout/reftests/font-matching/reftest.list
--- a/layout/reftests/font-matching/reftest.list	Tue Mar 01 17:27:27 2022 +0200
+++ b/layout/reftests/font-matching/reftest.list	Tue Mar 01 15:22:56 2022 +0000
@@ -172,5 +172,9 @@
 # Disabled because Osaka-Mono isn't available on automation.
 # skip-if(!cocoaWidget) == osaka-mono.html osaka-mono-ref.html
 
+# Try to use Wingdings font (assumed present on Windows and macOS)
+random-if(!winWidget&&!cocoaWidget) == wingdings-1.html wingdings-1-ref.html
+random-if(!winWidget&&!cocoaWidget) != wingdings-1.html wingdings-1-notref.html
+
 # Reset default prefs.
 defaults